src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/AddCacheWarmerPassTest.php

```diff
@@ -33,7 +33,6 @@ public function testThatCacheWarmersAreProcessedInPriorityOrder()
             ->will($this->returnValue($services));
         $container->expects($this->atLeastOnce())
             ->method('getDefinition')
-            ->with('cache_warmer')
```

Comment:
this removal does not look right to me. We don't want to return the same Definition mock for unrelated services